.github/workflows/deploy.yml

```diff
@@ -11,21 +11,28 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      
-      # Use GitHub Actions' cache to shorten build times and decrease load on servers
-      - uses: actions/cache@v2
+
+      - uses: actions/setup-python@v2

```

Comment:
I think python will already be installed, can u check?

Reply:
Yep, we can skip this.